src/notifications/generate.ts

```diff
@@ -145,6 +147,26 @@ export const generateNotificationMap: Record<
       .descriptionComment(ctx.comment)
       .targetPost(ctx.post, ctx.comment)
       .avatarManyUsers([ctx.commenter]),
+  post_mention: (
+    builder,
+    ctx: NotificationPostContext & NotificationDoneByContext,
+  ) =>
+    builder
+      .referencePost(ctx.post)
+      .icon(NotificationIcon.Comment)
+      .description((ctx.post as WelcomePost).content)
```

Comment:
content is too long here, shouldn't we truncate it or something?
Also to make it more generic please change the casting to FreeformPost

Reply:
Good point. I was thinking that this could already be truncated based on how long comments currently work. But will double-check that one.

Reply:
Yes you can apply the same mechanism for sure :)